### PR TITLE
Track world nomads widget with GA

### PR DESCRIPTION
**Pull request by tkali**
This one adds GA tracking for World Nomads widget we use on "Essential Information" and "Travel Insurance" pages.

### CODE REVIEW

**Comment by robuye** [comment]
:+1: if it works

File: app/assets/javascripts/lib/analytics/travel_insurance_omniture.js [hunk 1]
```diff
@@ -24,9 +24,29 @@ require([ "jquery", "lib/analytics/analytics" ], function($, Analytics) {
...
 
   // Before redirection (which the WN widget does, it's not a form submit)
-  // if the user clicked on the submit button, track click with Omniture
+  // if the user clicked on the submit button, track click with Omniture and GA
   window.onbeforeunload = function() {
     if (windowUnloadedFromSubmitClick) {
+
+      if (window.lp.analytics.api && window.lp.analytics.api.trackEvent) {
+
+        var destinations = $("#selected-destinations li").map(function() {
```
Comment (WunderBart) [inline app/assets/javascripts/lib/analytics/travel_insurance_omniture.js]: Can we use a `.js-` prefixed selector here?

File: app/assets/javascripts/lib/analytics/travel_insurance_omniture.js [hunk 2]
```diff
@@ -24,9 +24,29 @@ require([ "jquery", "lib/analytics/analytics" ], function($, Analytics) {
...
+          return $(this).text().replace(/[^a-zA-Z()\d\s:]/, "").trim();
+        }).get().join("|");
+
+        if ($(".js-travel-widget .input-validation-errors").length === 0) {
```
Comment (WunderBart) [inline app/assets/javascripts/lib/analytics/travel_insurance_omniture.js]: Same as above

File: app/assets/javascripts/lib/analytics/travel_insurance_omniture.js (hunk 1 above)
Comment (tkali) [inline app/assets/javascripts/lib/analytics/travel_insurance_omniture.js]: This is an external widget from World Nomads, we don't have control over it's internal structure.

File: app/assets/javascripts/lib/analytics/travel_insurance_omniture.js (hunk 2 above)
Comment (tkali) [inline app/assets/javascripts/lib/analytics/travel_insurance_omniture.js]: Same here, World Nomads' guts.

File: app/assets/javascripts/lib/analytics/travel_insurance_omniture.js (hunk 1 above)
Comment (WunderBart) [inline app/assets/javascripts/lib/analytics/travel_insurance_omniture.js]: Cool :+1:

File: app/assets/javascripts/lib/analytics/travel_insurance_omniture.js (hunk 2 above)
Comment (WunderBart) [inline app/assets/javascripts/lib/analytics/travel_insurance_omniture.js]: Also cool :smile:

Comment (tkali) [inline app/assets/javascripts/lib/analytics/travel_insurance_omniture.js]: Indeed sir.

**Comment by WunderBart** [comment]
:+1: if it works :trollface:

**Pull request merged**